src/modelfree/configs/multi/train.py

```diff
@@ -15,6 +16,90 @@
 TARGET_VICTIM = collections.defaultdict(lambda: 1)
 TARGET_VICTIM['multicomp/KickAndDefend-v0'] = 2
 
+HYPERPARAM_SEARCH_VALUES = {
+    'seed': tune.sample_from(
+        lambda spec: np.random.randint(1000)),
+
+    # Dec 2018 experiments used 2^11 = 2048 batch size.
+    # Aurick Zhou used 2^14 = 16384; Bansal et al use 409600 ~= 2^19.
+    'batch_size': tune.sample_from(
+        lambda spec: 2 ** np.random.randint(11, 16)),
+
+    'rl_args': {
+                # PPO2 default is 0.01. run_humanoid.py uses 0.00.
```

Comment:
Indentation here seems unnecsssarily large

Reply:
I think it's more readable when you can see the nested args on a more exaggerated indentation level, but I can change it to be less so